### DISCUSSION TITLE
git status

### DISCUSSION THREAD

**Discussion by molleweide** [cljoly/telescope-git-repo.nvim#18] [General]
i saw that you were working on entering git status directly. Are you doing this with `neogit`?

**Comment by cljoly** [comment]
No, I’m just using `:Telescope git_status`, since it’s shipped with telescope. But it’s not ideal, because if there are no changes, nothing is opened, so I’m kind of looking for something else without introducing more dependencies…

**Reply by molleweide** [reply]
Maybe have a default way of doing it and then option if eg. neogit exists then use that or another git plug.

**Reply by cljoly** [reply]
Yeah, that’s a good point.
